app/lib/katello/resources/cdn/katello_cdn.rb

```diff
@@ -36,12 +36,16 @@ def valid_path?(_path, _postfix)
           true
         end
 
+        def validate_cv_lce!
```

Comment:
wondering if can call this `validate!` instead and also include the checks for the organization like 
``` organization && content_view_id && lifecycle_environment_id```